packages/esm-tb-app/src/constants.ts

```diff
@@ -3,6 +3,18 @@ export const TbPatientTracingEncounterType = '98c938e9-fb3e-4982-ae22-0305cbd12f
 export const contactDate = '160753AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA';
 export const contactMethod = '166456AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA';
 
+// TB Cohorts
+export const clientsEnrolledForTb = '98fd11ba-cb4d-46f7-9b82-40d49949c7ef';
```

Comment:
@lucyjemutai when this code gets to working it will not work because of this `uuid`. I know he is on leave but @smallgod will have to check with @alaboso how are we managing this